TASVideos/Pages/Diagnostics/UserAgentInterventionReports.cshtml.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TASVideos.Pages.Diagnostics;
+
+[RequirePermission(PermissionTo.SeeDiagnostics)]
+public class UserAgentInterventionReportsModel : BasePageModel
+{
+	public void OnPost()
```

Comment:
You don't need this method

Reply:
Okay but how do I listen for POSTs though.